microlith/prometheus/alerting/couchbase/couchbase-rules.yaml

```diff
@@ -55,3 +55,39 @@ groups:
         title: "Disk Commit Fail on bucket: {{ $labels.bucket }}"
         description: A disk commit has failed on {{ $labels.bucket }} (cluster {{ $labels.cluster }}, node {{ $labels.instance }}).
         remediation: Review your infrastructure for signs of disk problems or any other issues, or contact Couchbase Technical Support.
+
+    - alert: CB90055-metadataOverhead-Warning
+      expr: |
+        (kv_total_memory_overhead_bytes / kv_ep_max_size) > 0.5 < 0.9
```

Comment:
Does this PromQL syntax work?

Reply:
yup, doesn't error and checked with different values for `kv_total_memory_overhead_bytes` (0 and 1000000000)